tests/console/validate_repos.pm

```diff
@@ -17,6 +17,8 @@ sub run {
 
     select_console 'root-console';
 
+    my %expected_repos = map { $_->{alias} => 1 } @{$test_data->{repos}};
+    my @actual_aliases = split(/\n/, script_output("zypper -n lr --uri | grep http | awk \'{print \$3}\'"));
```

Comment:
instead of `http` you might be looking for something like this:
`zypper lr | awk 'NR>4 {print $3}'`